.github/workflows/kcp-test.yml

```diff
@@ -1,3 +1,4 @@
+# Currently disabled in github actions
```

Comment:
Would prefer we remove the github actions file altogether

Reply:
I'm thinking if we are going back to support kcp in the future, we can still re-use this file. 
I have disable the github action already, so so leaving this file would not affect the ci.

Reply:
Yes, but can always go back in the Git history if we need to find it. Also, with how much churn occurs on the KCP API, it's likely this test won't be valid when/if we revisit KCP. I think I'd rather we remove it altogether to avoid any confusion.